frontend/src/components/Formats.tsx

```diff
@@ -15,6 +16,15 @@ export const Formats = {
   round: (value: number) => {
     return Math.round(value * 10) / 10
   },
+  chip: (value: ILicenseChip) => {
+    return (
+      <Chip
+        style={{ color: value.color, backgroundColor: value.background, fontFamily: 'Roboto Mono' }}
```

Comment:
could you use a className?

Reply:
Yes probably could - would have to pass properties to it.